README.md

```diff
@@ -122,6 +122,10 @@ models = [
   "Dlib", 
   "SFace",
   "GhostFaceNet",
+  "yolov8",
```

Comment:
what is YoloFace? if you set YoloFace to model_name, then it will raise exception because it expects yolov8, yolov11n, yolov11s or yolov11m.

Besides, all facial recognition model names are starting with uppercase.